tutorials/elixir-phoenix-on-google-app-engine.md

```diff
@@ -0,0 +1,183 @@
+    even using brunch to build and digest your assets.
+
+    Deployment will also take a few minutes to requisition and configure the
+    needed resources, especially the first time you deploy.
+
+3.  Once the deploy command has completed, you can run
+
+        gcloud app browse
+
+    to see your app running in production on App Engine.
+
+## Update your application
+
+Let's make a simple change and redeploy.
+
+1.  Open the front page template
+    `lib/appengine_example_web/templates/page/index.html.eex` in your editor.
+    Make a change to the HTML template.
+
+2.  Run the deployment command again:
+
+        gcloud app deploy
+
+    App Engine and the Elixir Runtime will take care of rebuilding your app,
+    deploying the updated version, and switching traffic from the old to the
+    new.
+
+3.  View your changes live by running
+
+        gcloud app browse
+
+## Clean up
+
+After you've finished this tutorial, you can clean up the resources you created
+on Google Cloud Platform so you won't be billed for them in the future. You can
+accomplish this either by deleting the project, or by stopping the App Engine
+service.
+
+### Deleting the project
+
+The easiest way to eliminate billing is to delete the project you created for
+the tutorial. To do so using gcloud:
+
+    gcloud projects delete <your-project-id>
+
+**Warning**: Deleting a project has the following consequences:
+
+If you used an existing project, you'll also delete any other work you've done
+in the project. You can't reuse the project ID of a deleted project. If you
+created a custom project ID that you plan to use in the future, you should
+delete the resources inside the project instead. This ensures that URLs that
+use the project ID, such as an appspot.com URL, remain available.
+
+### Stopping App Engine services
+
+To disable an App Engine service:
+
+1.  In the Cloud Platform Console, go to the
+    [App Engine Versions page](https://console.cloud.google.com/appengine/versions).
+2.  Make sure your project is selected. If necessary, pull down the project
+    selection dropdown at the top, and choose the project you created for this
+    tutorial.
+3.  If you deployed to a service other than "default", make sure it is selected
+    in the Service dropdown.
+4.  Select all the versions you wish to disable and click **Stop** at the top
+    of the page. This will free all of the Google Compute Engine resources used
+    for this App Engine service.
+
+## Next steps
+
+The [Elixir Samples](https://github.com/GoogleCloudPlatform/elixir-samples)
+repository contains a growing set of sample Elixir applications ready to deploy
+to Google Cloud, and examples of communicating with Google APIs from Elixir.
+
+See the [App Engine documentation](https://cloud.google.com/appengine/docs/flexible/)
+for more information on the features of App Engine, including scaling, health
+checks, infrastructure customization, and so forth.
```

Comment:
*for more information on App Engine features including scaling, health checks, and infrastructure customization.

Reply:
Done.